conf_general.h

```diff
@@ -190,7 +196,8 @@
 /*
  * Select default user motor configuration
  */
-//#include			"conf_test.h"
+#include			"mcconf_default.h"
```

Comment:
The default configuration should not be selected from here. It will be selected later if it isn't overridden here or somewhere else.